backend/fittoring/src/main/java/fittoring/mentoring/business/service/PhoneVerificationFacade.java

```diff
@@ -0,0 +1,23 @@
+package fittoring.mentoring.business.service;
+
+import fittoring.mentoring.business.model.Phone;
+import fittoring.mentoring.infra.SmsRestClientService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class PhoneVerificationFacade {
+
+    private static final String VERIFICATION_MESSAGE_PREFIX = "핏토링 인증번호는 [";
+    private static final String VERIFICATION_MESSAGE_SUFFIX = "] 입니다.";
+
+    private final PhoneVerificationService phoneVerificationService;
+    private final SmsRestClientService smsRestClientService;
+
+    public void sendPhoneVerificationCode(String phoneNumber) {
+        Phone phone = new Phone(phoneNumber);
+        String code = phoneVerificationService.createPhoneVerification(phone);
+        smsRestClientService.sendSms(phone, VERIFICATION_MESSAGE_PREFIX + code + VERIFICATION_MESSAGE_SUFFIX);
```

Comment:
p2: 전송될 메세지의 형식을 여기서 지정해주면 몇가지 우려되는 상황이 있을 것 같습니다.
- `VERIFICATION_MESSAGE_PREFIX + code + VERIFICATION_MESSAGE_SUFFIX`의 조합을 강제할 수 없다.
- 메세지 형식의 변경 책임이 해당 객체에서 일어나면 파사드 객체가 비즈니스 로직에 관여하게 됩니다.

기존 `SmsMessageFormatter` 에 인증번호 포멧팅을 담당하는 메서드를 추가하여 메세지 형식 생성 책임을 이동시키는건 어떨까요?

개선 방법 예시
```java
public class SmsMessageFormatter {
...
public String createSmsVerificationMessage(String code) {
        return String.format("핏토링 인증번호는 [%s] 입니다.", code);
    }
}
```

```suggestion
    private final PhoneVerificationService phoneVerificationService;
    private final SmsRestClientService smsRestClientService;

    public void sendPhoneVerificationCode(String phoneNumber) {
        Phone phone = new Phone(phoneNumber);
        String code = phoneVerificationService.createPhoneVerification(phone);
        String message = smsMessageFormatter.createSmsVerificationMessage(code);
        smsRestClientService.sendSms(phone, message);
```

Reply:
좋습니다. 반영했습니다!